### PR TITLE
blog post about march 15 2022 community feedback session

### PR DESCRIPTION
**Pull request by lauriemerrell**
Adds blog post about March 15 2022 community feedback session, including two screenshots of the Miro board.

### CODE REVIEW

File: _posts/blog/2022-04-26-march-2022-community-feedback.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+image: /images/blog/2022-04-26-march-2022-community-feedback/20220315_community_feedback_miro_board.png
+author: Laurie Merrell
+author_url: https://twitter.com/laurie_merrell
+author_image: /images/people/board/laurie_merrell.jpg
+author_bio: "Board Member, Chi Hack Night"
+author2: 
+author2_url:
+author2_bio:
+published: true
+featured: false
```
Comment (derekeder) [inline _posts/blog/2022-04-26-march-2022-community-feedback.md]: to have the blog post show up on the home page, lets set `featured: true`

Comment (derekeder) [reply _posts/blog/2022-04-26-march-2022-community-feedback.md]: we should also make the two currently featured blog posts no longer featured as they're pretty old now, so this one will be the only one featured on the home page

**Review by derekeder — CHANGES_REQUESTED** [review]
looks great! 

In addition to the comments about featuring on the home page, the two images for this post are pretty large (1-2mb). I try to make them ~200k or less to help with page loading speeds. The best ways to do this are to resize (1600px wide should be enough) and save in a more compressed format like `jpg`.

File: _posts/blog/2022-04-26-march-2022-community-feedback.md (hunk 1 above)
Comment (lauriemerrell) [reply _posts/blog/2022-04-26-march-2022-community-feedback.md]: Thanks Derek, resolved in fe048c934e78f9a5c62376d198ee161d4d406162 and 9ed131ed1ee3d756e1c406751356d53d84e0d2bf

File: _posts/blog/2022-04-26-march-2022-community-feedback.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
+---
+layout: blog
+categories: 
+  - blog
+title: "March 2022 Community Feedback Session"
+description: "On March 15, 2022, Chi Hack Night held a hybrid community feedback session
+to assess how our community is feeling about the current phase of the pandemic and what people
+are looking for from Chi Hack Night in this moment."
+date: 2022-04-26
+image: /images/blog/2022-04-26-march-2022-community-feedback/20220315_community_feedback_miro_board.png
```
Comment (derekeder) [inline _posts/blog/2022-04-26-march-2022-community-feedback.md]: files are now `jpg` so we need to change this from `png`

File: _posts/blog/2022-04-26-march-2022-community-feedback.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+published: true
+featured: true
+---
+
+<!---
+use figcaption instead of manually formatting caption
+from: https://towardsdev.com/3-ways-to-add-a-caption-to-an-image-using-markdown-f2ca30562be6
+-->
+
+<figure><img src="/images/blog/2022-04-26-march-2022-community-feedback/20220315_community_feedback_miro_board.png" alt="A screenshot of a Miro board. There are a set of four panels with discussion prompt headers and a variety of colorful sticky notes with responses below." style="width:100%"><figcaption align = "center">Responses from the March 15, 2022 community feedback session</figcaption></figure>
```
Comment (derekeder) [inline _posts/blog/2022-04-26-march-2022-community-feedback.md]: `png` => `jpg`

File: _posts/blog/2022-04-26-march-2022-community-feedback.md [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+
+4. **What speakers would you like to see at Chi Hack Night? What organizations do you think the Chi Hack Night community could benefit hearing from?**
+* Attendees proposed a variety of government, media, and advocacy organizations, which have been shared with the booking committee. 
+
+<figure><img src="/images/blog/2022-04-26-march-2022-community-feedback/20220315_community_feedback_proposed_speakers_miro.png" alt="A screenshot of a Miro board. There is a prompt at the top reading: 'What speakers would you like to see at Chi Hack Night? What organizations do you think the Chi Hack Night community could benefit hearing from?'. There are a variety of colorful sticky notes below the header, with responses like 'Cook County Government' and 'Better Streets Chicago'." style="width:100%"><figcaption align = "center">Speaker suggestions from feedback session attendees</figcaption></figure>
```
Comment (derekeder) [inline _posts/blog/2022-04-26-march-2022-community-feedback.md]: `png` => `jpg`

**Review by derekeder — CHANGES_REQUESTED** [review]
looking at the deploy preview (https://deploy-preview-428--chihacknight-website.netlify.app/) the images aren't loading. The url reference needs to be updated with the new `jpg` file extension

File: _posts/blog/2022-04-26-march-2022-community-feedback.md (hunk 4 above)
Comment (lauriemerrell) [reply _posts/blog/2022-04-26-march-2022-community-feedback.md]: Ope thanks

File: _posts/blog/2022-04-26-march-2022-community-feedback.md (hunk 2 above)
Comment (lauriemerrell) [reply _posts/blog/2022-04-26-march-2022-community-feedback.md]: Thanks 🙏

**Review by derekeder — APPROVED** [review]
looks great! go ahead and merge and it will automatically deploy

**Pull request merged**